python/ray/autoscaler/_private/load_metrics.py

```diff
@@ -354,6 +352,8 @@ def info_string(self):
         )
 
     def _info(self):
+        import numpy as np
```

Comment:
Can you just remove the `np.` parts of these `np.mean`, `np.min`, and `np.max` so that we can just delete this instead?